redis/cluster.py

```diff
@@ -1637,7 +1636,7 @@ def __init__(
         cluster_response_callbacks=None,
         startup_nodes=None,
         read_from_replicas=False,
-        cluster_error_retry_attempts=3,
+        cluster_error_retry_attempts=5,
```

Comment:
Maybe this should be configureable.  It seems likely that a user will want to specify their retries - but with a sane default of course.

Reply:
It is configurable - the user can pass 'cluster_error_retry_attempts' when creating a new RedisCluster instance.
In the tests I ran 3 attempts wasn't enough to handle some of the failover scenarios. obviously it depends on the socket_timeout a user will configure but 5 retries seems sane to me. what do you think?